src/index.js

```diff
@@ -336,7 +336,8 @@ function removeNodeFromHost(host, node) {
 }
 
 function removeSlotFromRoot(root, node) {
-  node.assignedNodes().forEach(slotNodeFromSlot);
+  const assignedNodes = Array.prototype.slice.call(node.assignedNodes());
```

Comment:
A more direct way would be to `Array.prototype.forEach.call(node.assignedNodes(), slotNodeFromSlot);`.